enhancements/monitoring/alerts-ui-management.md

```diff
@@ -0,0 +1,337 @@
+---
+title: alerts-ui-managment
+authors:
+  - "@sradco"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2025-07-24
+last-updated: 2025-07-30
+tracking-link:
+  - "https://issues.redhat.com/browse/CNV-46597"
+  - "https://issues.redhat.com/browse/CNV-62972"
+---
+# New APIs to support Alerts UI Management in OpenShift
+
+## Summary
+Provide a user‑friendly UI and REST API for defining, viewing, editing, disabling/enabling and silencing Prometheus alerts without manual YAML edits, reducing alert fatigue and improving operational efficiency.
+Platform alerts will be updated via entries in the `AlertRelabelConfig` CR (in `cluster-monitoring-config`) rather than editing their original `PrometheusRule`.
```

Comment:
this sentence is a bit fuzzy because there's no AlertRelabelConfig in cluster-monitoring-config. I assume that we want to say that platform alerts will be overriden leveraging the AlertRelabelConfig and AlertingRule CRDs.